doc/source/report/plots/plot_distribution.py

```diff
@@ -0,0 +1,14 @@
+import matplotlib.pyplot as plt
+from mousestyles import data
+from mousestypes.est_power_param import fit
```

Comment:
@lunadadada  This is invalid.  I will fix it for you in another pull request.  Please test your code in the future, I shouldn't have to fix your typos for you.